frontends/relay/relay_visitor.py

```diff
@@ -187,6 +238,17 @@ def visit_call(self, call) -> tvm.relay.Call:
         """
         # Visit the call arguments.
         call.args = [self.visit(a) for a in call.args]
+        # dealing w/ the fact that visit_var returns list
```

Comment:
Nit: perhaps put this in its own helper function with a comment explaining what it does?